packages/components/src/internal/util/Date.ts

```diff
@@ -158,13 +174,19 @@ export function parseFNSTimeFormat(timePart: string): string {
 // format (e.g. it supports formats similar to "yyyy-MM-dd hh:mm" or "yyyy-MM-dd hh:mm a").
 export function parseDateFNSTimeFormat(dateFormat: string): string {
     if (!dateFormat) return undefined;
-    const parts = dateFormat.split(' ');
-
-    if (parts.length > 1) {
-        return parseFNSTimeFormat(parts[1]);
+    let splitIndex = dateFormat.indexOf(' ');
+
+    let format;
+    if (splitIndex > -1) {
+        const remaining = dateFormat.substring(splitIndex + 1);
+        format = parseFNSTimeFormat(remaining);
+        if (!format && remaining.indexOf(' h') > 0) {
```

Comment:
I must be reading this wrong. Hasn't the space before the 'h' already been removed via the substring starting at `splitIndex+1`?

Reply:
This is for a special case when ' ' is used as delimiter for the date part, for example: yyyy MM dd hh:mm. i'll add a comment